README.md

```diff
@@ -214,7 +216,7 @@ class MyStorage
 end
 
 Fortnox::API.configure do |config|
-  config.storages = {
+  config.token_stores = {
     default: MyStorage.new,
     another_account: MyStorage.new(account: :another_account)
```

Comment:
Why is the `(account: :anothet_account)` needed? You are already keeping them under keys in the token_store hash. Each one is an independent instance of a store so ...

Reply:
It's for the Redis part in the token store: we set `@prefix = account.empty? ? :default : "#{account}"` and `access_token_key` is then `"#{@prefix}_access_token"`. If we don't pass the `account` parameter to the second instance of the token store, the stores will overwrite each others keys since the Redis instance is assumed to be shared between the token stores.

Reply:
I will rename `MyStorage` to `MyTokenStore` though since we have renamed `storages` to `token_stores`.